llvm/lib/Target/AMDGPU/AMDGPUResourceUsageAnalysis.cpp

```diff
@@ -112,11 +112,15 @@ bool AMDGPUResourceUsageAnalysis::runOnModule(Module &M) {
 
   // By default, for code object v5 and later, track only the minimum scratch
   // size
+  uint32_t AssumedStackSizeForDynamicSizeObjects =
+      clAssumedStackSizeForDynamicSizeObjects.getDefault().getValue();
+  uint32_t AssumedStackSizeForExternalCall =
+      clAssumedStackSizeForExternalCall.getDefault().getValue();
```

Comment:
I don't understand why we need the `.getDefault()` -- wouldn't this make us ignore the value set in cli?